Kentico.Kontent.Delivery.Extensions.Autofac.DependencyInjection.Tests/ServiceCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Kentico.Kontent.Delivery.Abstractions;
+using Kentico.Kontent.Delivery.Caching;
+using Microsoft.Extensions.DependencyInjection;
+using Kentico.Kontent.Delivery.Extensions.Autofac.DependencyInjection.Extensions;
+using System;
+using Xunit;
+using FluentAssertions;
+using Autofac;
+using FakeItEasy;
+
+namespace Kentico.Kontent.Delivery.Extensions.Autofac.DependencyInjection.Tests
+{
+    public class ServiceCollectionExtensionsTests
```

Comment:
can we add a test testing the multi-`ITypeProvider` scenario?

Reply:
I do.